npm/private/npm_package.bzl

```diff
@@ -224,11 +221,13 @@ def _impl(ctx):
         targets = ctx.attr.data,
     ))
 
-    copy_to_directory_action(
+    copy_to_directory_bin_action(
```

Comment:
nit: would it read better for bazel-lib to expose a factory function for creating actions?

Reply:
hmmm. not sure what you mean. this function is a wrapper around ctx.actions.run: https://github.com/aspect-build/bazel-lib/blob/fe867981ee5606415a0b1656ee9e034fb89924c7/lib/private/copy_to_directory.bzl#L618